### PR TITLE
add ability to create an account password policy

### PR DESCRIPTION
**Pull request by ninjatux**
This PR add a proxy to the resource `aws_iam_account_password_policy`.

Fixes:

* https://github.com/nearform/covid-tracker-infrastructure/issues/367
* https://github.com/nearform/covid-tracker-infrastructure/issues/366

### CODE REVIEW

**Comment by ninjatux** [comment]
where do you think we should test this? in `nfcs-dev` is tricky for the SSO setup available there

File: versions.tf [hunk 1]
```diff
@@ -1,10 +1,6 @@
 terraform {
   required_version = ">= 0.12.29, < 0.14"
 
-  # Leaving this, even though we have moved towards using this repo as a module - will ignore in that case
```
Comment (pmcgrath) [inline versions.tf:4]: Removing this probably should require doc updates - will be misleading

File: variables.tf [hunk 2]
```diff
@@ -20,6 +20,65 @@ variable "profile" {
...
+  description = "Whether to require numbers for user passwords"
+  type        = bool
+  default     = null
+}
+variable "account_password_require_symbols" {
+  description = "Whether to require symbols for user passwords"
+  type        = bool
+  default     = null
+}
+variable "enforce_mfa" {
```
Comment (pmcgrath) [inline variables.tf:76]: Should we remove this var https://github.com/nearform/covid-green-infra/blob/dev/variables.tf#L26
and change the admins.tf file to use the new enforce_mfa var ?

File: operators.tf [hunk 3]
```diff
@@ -92,18 +92,147 @@ data "aws_iam_policy_document" "operators" {
...
 }
 
 resource "aws_iam_group_policy" "operators" {
   group  = aws_iam_group.operators.id
-  name   = "${module.labels.id}-operators"
+  name   = format("%s-%s", module.labels.id, "operators")
   policy = data.aws_iam_policy_document.operators.json
 }
 
+resource "aws_iam_group_policy" "operators_mfa" {
```
Comment (pmcgrath) [inline operators.tf:229]: Since operators are a group as opposed to a role does this work - don't know

File: operators.tf [hunk 4]
```diff
@@ -92,18 +92,147 @@ data "aws_iam_policy_document" "operators" {
...
+      "sts:GetSessionToken"
+    ]
+    resources = [
+      "*"
+    ]
+    condition {
+      test     = "BoolIfExists"
+      variable = "aws:MultiFactorAuthPresent"
+      values = [
+        "false",
```
Comment (pmcgrath) [inline operators.tf:212]: should this be "true" ? Not sure

File: operators.tf (hunk 3 above)
Comment (pmcgrath) [reply operators.tf:229]: should this only be attached if mfa is enabled ?

**Review by pmcgrath — COMMENTED** [review]
Will need a change log entry

File: operators.tf (hunk 4 above)
Comment (dgonzalez) [reply operators.tf:212]: I think it is `false` the right one!

**Comment by dgonzalez** [comment]
@daynelucas this one looks good to me but we need to test it in depth. Let's do some testing in QA (or dev) and check that it makes sense.

**Comment by daynelucas** [comment]
@rgormley You mentioned doing something similar in standup today

**Pull request closed without merging**